src/main/java/frc/trigon/robot/subsystems/collector/Collector.java

```diff
@@ -0,0 +1,45 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Collector extends SubsystemBase {
+    private static final WPI_TalonFX
+            openingMotor = CollectorConstants.OPENER_MOTOR,
+            collectingMotor = CollectorConstants.COLLECTOR_MOTOR;
+
+    private final static Collector INSTANCE = new Collector();
+
+    private Collector() {
+
+    }
+
+    public static Collector getInstance() {
+        return INSTANCE;
+    }
+
+    private void collect() {
+        openingMotor.set(CollectorConstants.OPENING_POWER);
+        collectingMotor.set(ControlMode.PercentOutput, CollectorConstants.COLLECTING_POWER);
```

Comment:
`ControlMode.PercentOutput` is unnecessary as that is the default

Reply:
more clarity and consistency

Reply:
No. `.set` is always a percent output there is nothing to clarify

Reply:
If you use `set()` with a `ControlMode` then it makes sense to keep using the same format.

Reply:
We have never put a control mode for percentage output. its unnecessary. so unless you want to change it in every branch for no reason then we are leaving it without the unnecessary control mode

Reply:
It's just more readable. When you see set() with a control mode of position, if you see after that set() without it, it's not super clear, and you'll have to read the documentation.

Reply:
What are you talking about? everyone knows how `.set` works... it's pretty intuitive that it's a percent